pkg/minikube/node/start.go

```diff
@@ -706,8 +706,8 @@ func getPort() (int, error) {
 }
 
 // startHostInternal starts a new minikube host using a VM or None
-func startHostInternal(api libmachine.API, cc *config.ClusterConfig, n *config.Node, delOnFail bool) (*host.Host, bool, error) {
```

Comment:
how about  keep the original signature here
```
func startHostInternal(api libmachine.API, cc *config.ClusterConfig, n *config.Node, delOnFail bool) (*host.Host, bool, error)
```

and just pass "options.DeleteOnFailure" to the caller func 

```
hostInfo, preExists, err = startHostInternal(m, cfg, node, options.DelOnFailure)
```

its okay if do pass the full run.options if it is being passed down to another func, but otherwise lets just call the func with as Little information as it needs.

Reply:
We do pass the value to next function, see bellow.